index.d.ts

```diff
@@ -269,7 +269,7 @@ declare namespace RNRF {
     jump(props: props): void,
     refresh(props: props): void,
     focus(props: props): void,
-    create(scene: Scene, wrapBy?: () => any): Object
+    create(scene: React.ReactNode, wrapBy?: () => any): Object
```

Comment:
https://github.com/xirc/react-native-router-flux-sandbox is the example.
'master' branch uses 69e0ab4
'fixed-one' branch uses a545cad